src/status_im/ui/screens/anonymous_metrics_settings/views.cljs

```diff
@@ -0,0 +1,312 @@
+(ns status-im.ui.screens.anonymous-metrics-settings.views
+  (:require-macros [status-im.utils.views :refer [defview letsubs]])
+  (:require [re-frame.core :as re-frame]
+            [status-im.ui.components.animation :as animation]
+            [status-im.react-native.resources :as resources]
+            [status-im.anon-metrics.core :as anon-metrics]
+            [status-im.ui.components.accordion :as accordion]
+            [status-im.ui.components.react :as react]
+            [status-im.ui.components.colors :as colors]
+            [status-im.ui.components.icons.icons :as icons]
+            [status-im.i18n.i18n :as i18n]
+            [quo.design-system.spacing :as spacing]
+            [status-im.ui.components.topbar :as topbar]
+            [quo.core :as quo]))
+
+(defonce rules-github-href "https://github.com/status-im/status-go/blob/develop/_docs/app-metrics.md")
+
+(defn close-learn-more-modal []
```

Comment:
imo its kinda redundant code , it just hides event, so when you read code which uses this function you don't know what it's doing, and you need to find it to look or modify, why don't just directly use dispatch?

Reply:
It's a leftover from something I was trying. Direct dispatches are better, will fix it.